### ISSUE TITLE
Closing Drawer automatically

### ISSUE THREAD

**Issue by Husky22** [lambdalisue/fern.vim#94]
Is there a way to close the drawer automatically after open:edit a file? 
I don't want the bookmark scheme to be open constantly.

**Comment by lambdalisue** [comment]
Unfortunately, it's not possible with current implementation. PR is welcome 👍

**Comment by lambdalisue** [comment]
@Husky22 #113 solves your problem I think. Could you try?

**Comment by Husky22** [comment]
@lambdalisue 
At first thanks for taking your time on this problem!
I tried it in my NVim configuration and it seems to collide with [vim-which-key](https://github.com/liuchengxu/vim-which-key) somehow.
I get the error message
 `<space> : is undefined`
In vanilla NVim it works as expected.

Just to be clear my settings are:
```nvim
function! s:init_fern() abort
    nmap <buffer> q :<C-u>FernDo close<CR>
    nmap <buffer> <C-Space> <Plug>(fern-action-mark-toggle)
    nmap <buffer><silent> <Plug>(fern-my-open-and-close)
                \ <Plug>(fern-action-open)
                \ :<C-u>FernDo close<CR>
    nmap <buffer><silent> go <Plug>(fern-my-open-and-close)  
endfunction


augroup fern-custom
  autocmd! *
  autocmd FileType fern call s:init_fern()
augroup END
```

I don't know whether this is now a problem of vim-which-key or a problem on behalf of Fern.

`<space>` is my leader key and Fern is called via vim-which-key.
Creating a keymap to call fern without vim-which-key did not resolve the issue either.
Sadly I could not find any further information in my logs except the already stated error message.

As of your documentation I also don't understand the meaning of the second colon in the command `:FernDo :`

**Comment by lambdalisue** [comment]
Try `nnoremap` instead. I'm not sure but vim-which-key might map `:`.

```vim
nnoremap <buffer> q :<C-u>FernDo close<CR>

" Or create <Plug>(my-fern-close) or whatever by nnoremap first
nnoremap <buffer> <Plug>(my-fern-close) :<C-u>FernDo close<CR>
```

> As of your documentation I also don't understand the meaning of the second colon in the command :FernDo :

Executing `:` does nothing usually. Try `:execute ":"`.

**Comment by lambdalisue** [comment]
Closed by #113

**Comment by Husky22** [comment]
Just to give my current solution for this problem for everyone using Fern and Vim-Which-Key:
The following line solves the Problem of calling Which Key after opening:
```vim
nmap <silent><buffer> <CR> <Plug>(fern-action-open)<ESC><BAR>:FernDo close<CR>
```
